### ISSUE TITLE
ANDI/VO: Sorting buttons are focusable twice on the domain request table

### ISSUE THREAD

**Issue by zandercymatics** [cisagov/getgov#3050]
### Issue description

Currently, the sorting buttons on the domain request table are focusable twice (the button, then the button icon). As such, the VO is reading duplicative content. 

### Acceptance criteria

- [x] Each sort button is only accessible once via keyboard navigation.
- [x] ANDI and VO do not show duplicate aria labels and read the content once.
- [x] Both ACs are resolved on both the org view AND the non-org view

### Additional context

[Slack thread](https://cisa-corp.slack.com/archives/C07TZN9MQ9Z/p1730844423712669)

INTERMITTENT ISSUES FOUND (Need further testing):
- Search button works intermittently with keyboard command
-The <div> group for "Domain Requests is not focusable and skips over the "Domain Requests" header and subheader text. This removes important content for users relying on screen readers.
-When clearing out the search bar, the action is done but no focus is available

### Links to other issues

- 🔄 #2594 (originating issue)
